.travis.yml

```
@@ -0,0 +1,100 @@
+language: python
+branches:
+  only:
+  - 12.0
+  - /^st.*$/
+
+cache:
+  directories:
+    - $HOME/.cache/pip
+    # Add cache for pre-commit on repo that uses it (mandatory for >= 13)
+    - $HOME/.cache/pre-commit
+
+python:
+  # For branches <= 10.0, put `- "2.7.13"`
+  # For branches 11.0 and 12.0, put `- "3.5"`
+  - "3.5"
+
+addons:
+  # By default postgresql-9.1 is installed but there is issue related:
+  #  https://github.com/OCA/maintainer-quality-tools/issues/432#issuecomment-281580935
+  # Better use higher PostgreSQL version
+  postgresql: "9.6"
+  apt:
+#    sources:
+#    Search your sources alias here:
+#      https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json
+    packages:
+      - expect-dev # provides unbuffer utility
+#     Search your packages here:
+#       https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise
+#     For wkhtmltopdf, see the env section below
+
+# Sometimes complicated website repos need Compass & SaSS:
+before_install:
+  - echo -e "machine github.com\n  login $CI_USER_TOKEN" > ~/.netrc
+#  - rvm install ruby --latest
+#  - gem install bootstrap-sass
+#  - gem install compass --pre
+
+stages:
+  - linting
+  - test
+
+jobs:
+  include:
+    # use this linting job repos that use pre-commit (mandatory for >= 13)
+    # - stage: linting
+    #   name: "pre-commit"
+    #   python: "3.6"  # Black requires python >= 3.6
+    #   install: pip install pre-commit
+    #   script: pre-commit run --all --show-diff-on-failure --verbose --color always
+    #   after_success:
+    #   before_install:
+    # use this linting job for repos that do not use pre-commit (default for < 13)
+    # - stage: linting
+    #   env:
+    #     - LINT_CHECK="1"
+    - stage: test
+      env:
+        # add MAKEPOT="1" to a TEST line to export .pot files from
+        # the test database after test success
+        - TESTS="1" ODOO_REPO="OCA/OCB"
```

```suggestion
        - TESTS="1" ODOO_REPO="odoo/odoo"
```
Keep it consistent with the other RPL repos.